nest/src/AggregateToken.sol

```diff
@@ -143,6 +163,18 @@ contract AggregateToken is ComponentToken, IAggregateToken, ERC1155Holder {
         return super.asset();
     }
 
+    /// @inheritdoc IComponentToken
+    function deposit(
+        uint256 assets,
+        address receiver,
+        address controller
+    ) public override(ComponentToken, IComponentToken) returns (uint256 shares) {
+        if (_getAggregateTokenStorage().paused) {
+            revert DepositPaused();
+        }
+        return super.deposit(assets, receiver, controller);
+    }
+
     /// @inheritdoc IComponentToken
     function redeem(
```

Comment:
This should be the same as the pre-deposit vault where we don't want to accept more than a certain amount of deposits, but we don't want to stop redemptions because that could lead to FUD if people can't withdraw.